test/linalg/symmetric.jl

```diff
@@ -3,36 +3,37 @@
 using Base.Test
 
 srand(101)
-debug = false #Turn on for more debugging info
 
-#Pauli σ-matrices
-for σ in map(Hermitian, Any[ eye(2), [0 1; 1 0], [0 -im; im 0], [1 0; 0 -1] ])
+@testset "Pauli σ-matrices" for
+    σ in map(Hermitian, Any[ eye(2), [0 1; 1 0], [0 -im; im 0], [1 0; 0 -1] ])
```

Comment:
yes, just commenting in the existing thread - your most recent version fixes this one but does what I was complaining about here in 3 other places